### DISCUSSION TITLE
When can it be compatible with native SQLite that supports the new version of Nodejs

### DISCUSSION THREAD

**Discussion by aidenconst** [sequelize/sequelize#17605] [Help & Questions]
When can it be compatible with native SQLite that supports the new version of Nodejs

**Comment by rzl** [comment]
I also want to know, I'm tired of npm install.
how to use sequlize with mysql:sqlite?
https://nodejs.org/docs/latest-v22.x/api/sqlite.html

**Comment by WikiRik** [comment]
We're using https://github.com/TryGhost/node-sqlite3 internally which bundles sqlite itself. Using custom adapters is something that we are working on but is not planned for the upcoming major release. Don't expect support to be added to sequelize anytime soon, especially with the feature still being experimental in Node
